### PR TITLE
Bug/fromobj

### PR DESCRIPTION
**Pull request by matiu**
- bump to 8.11
- upgrades `new Buffer` in mnemonics
- fixes (toObj) in BWS (not used at the moment)

### CODE REVIEW

File: packages/bitcore-wallet-client/src/lib/credentials.ts [hunk 1]
```diff
@@ -132,6 +134,21 @@ export class Credentials {
...
   }
+
+  /* 
+   * creates an ERC20 wallet from a ETH wallet
+   */
+  getTokenCredentials(token: { name: string, symbol: string, address:string }) {
+    const ret = _.cloneDeep(this);
+    ret.walletId = `${ret.walletId}-${token.address}`;
+    ret.coin = token.symbol.toLowerCase();
+    ret.walletNane = token.name;
```
Comment (justinkook) [inline packages/bitcore-wallet-client/src/lib/credentials.ts]: walletName

File: packages/bitcore-wallet-client/src/lib/credentials.ts [hunk 2]
```diff
@@ -132,6 +134,21 @@ export class Credentials {
 
     return x;
   }
+
+  /* 
+   * creates an ERC20 wallet from a ETH wallet
+   */
+  getTokenCredentials(token: { name: string, symbol: string, address:string }) {
```
Comment (justinkook) [inline packages/bitcore-wallet-client/src/lib/credentials.ts]: `address:string` needs space between

File: packages/bitcore-wallet-client/src/lib/credentials.ts [hunk 3]
```diff
@@ -63,14 +64,15 @@ export class Credentials {
   use145forBCH: any;
   addressType: string;
   keyId: string;
-
+  token: string;
```
Comment (justinkook) [inline packages/bitcore-wallet-client/src/lib/credentials.ts]: `token?: string` will not always be available for every wallet.

Comment (matiu) [reply packages/bitcore-wallet-client/src/lib/credentials.ts]: https://github.com/bitpay/bitcore/pull/2562

File: packages/bitcore-wallet-client/src/lib/credentials.ts (hunk 1 above)
Comment (matiu) [reply packages/bitcore-wallet-client/src/lib/credentials.ts]: https://github.com/bitpay/bitcore/pull/2562

File: packages/bitcore-wallet-client/src/lib/credentials.ts (hunk 2 above)
Comment (matiu) [reply packages/bitcore-wallet-client/src/lib/credentials.ts]: we are using 
```
> tslint --fix -c tslint.json 'src/**/*.ts'
```
and it does not force that :/

**Pull request merged**